Sources/Public/Animation/LottieView.swift

```diff
@@ -344,10 +386,7 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
   private let placeholder: (() -> Placeholder)?
 
   private func loadAnimationIfNecessary() {
-    guard
-      let loadAnimation = loadAnimation,
-      animationSource == nil
```

Comment:
I think this change means we'll re-call the `loadAnimation` every time `onAppear` is called, which I don't think we want. Perhaps instead we should change the `valueChanged` block to:

```
.valueChanged(value: loadAnimationTrigger.wrappedValue) { _ in
  animation = nil
  loadAnimationIfNecessary()
}
```

This additionally makes it so updating the `loadAnimationTrigger` puts the view back in a loading state, which may or may not be desirable. I do think this makes sense though, since we've effectively changed the animation data source to no longer refer to the previous animation.

Reply:
@calda I followed your recommendation. Looks like we do show the placeholder view after switching animations. The GIF in the PR description is updated with how it looks like after the changes